drivers/ieee802154/Kconfig.nrf5

```diff
@@ -42,4 +42,12 @@ config IEEE802154_NRF5_EXT_IRQ_MGMT
 	  the system. One example of external radio IRQ provider could be
 	  a radio arbiter used in dynamic multiprotocol applications.
 
+config IEEE802154_NRF5_VENDOR_OUI
```

Comment:
I can move it, as you suggest, to general drivers Kconfig and not assign default value to it. Thanks to this it will be possible to check in specific driver files if value was set by someone and that value should be used or it wasn't and for example some default, driver defined value can be set. Do you find it being useful for other drivers? Main use case is to have some constant value, specific for given driver and use it in most cases, but also to not close the doors for some vendors and users, which might want to set own value. I think that such approach doesn't give much benefit, as with generalization of this config, each driver will have to define own default value anyway. Finding the best default value for all drivers is not possible, as this value defines specific vendor and it can't be general. Please consider also that setting OUI at the configuration level might be valid for some other drivers, but not necessarily for all, as some chips have their addresses set in the factory and it's not possible to change them. I understand your statement that all drivers should work in the same way, so please verify if described solution would be better in your opinion.

Reply:
If there is no objection, I do prefer define IEEE802154_VENDOR_OUI at Kconfig. As far I understand, you can silent define your own default OUI at Kconfig.nrf5, for instance, since symbol now in 15.4 scope. User always have the possibility to override Kconfig symbols.

I'm not sure if checking !defined(IEEE802154_VENDOR_OUI) is the best approach. Because if in the future we define a default value the logic will be broken. Maybe ignore vendor OUI if value is Zero could be more reliable. Remember that you can override OUI at any place. Because of that I think is better to define a default value.